### PR TITLE
chore: add verification level to config

### PR DESCRIPTION
**Pull request by quake**
This PR adds verification level to the config file, some integration test needs this feature.

### CODE REVIEW

File: chain/src/tests/util.rs [hunk 1]
```diff
@@ -48,7 +48,11 @@ pub(crate) fn start_chain(
 
     let notify = NotifyService::default().start::<&str>(None);
     let chain_service = ChainBuilder::new(shared.clone(), notify)
-        .verification(verification)
+        .verification_level(if verification {
+            VerificationLevel::Full
+        } else {
+            VerificationLevel::None
+        })
```
Comment (keroro520) [inline chain/src/tests/util.rs]: Why not change `verification: bool` to `CheckLevel` ?

File: src/subcommand/run.rs [hunk 2]
```diff
@@ -34,7 +34,11 @@ pub fn run(args: RunArgs, version: Version) -> Result<(), ExitCode> {
 
     let notify = NotifyService::default().start(Some("notify"));
 
-    let chain_controller = setup_chain(shared.clone(), notify.clone());
+    let chain_controller = setup_chain(
+        shared.clone(),
+        notify.clone(),
+        args.config.verification_level,
```
Comment (keroro520) [inline src/subcommand/run.rs]: FYI. I prefer to pass `args.config` to `setup_chain` instead.

File: util/app-config/src/app_config.rs [hunk 3]
```diff
@@ -38,6 +39,7 @@ pub enum AppConfigContent {
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct CKBAppConfig {
     pub data_dir: PathBuf,
+    pub verification_level: Option<VerificationLevel>,
```
Comment (keroro520) [inline util/app-config/src/app_config.rs]: Move it into `ChainConfig` ?

**Comment by quake** [comment]
> It's not a good idea to make verification level configurable. We should find other ways to make writing integration tests more convenient.

New PR: https://github.com/nervosnetwork/ckb/pull/732

**Pull request closed without merging**